include/n64sys.h

```diff
@@ -216,6 +216,18 @@ inline volatile unsigned long get_ticks_ms(void)
 void wait_ticks( unsigned long wait );
 void wait_ms( unsigned long wait_ms );
 
+/**
+ * @brief Force a complete halt of all processors
+ *
+ * @note It should occur whenever a reset has been triggered 
+ * and its past its RESET_TIME_LENGTH grace time period.
+ * This function will shut down the RSP and the CPU, blank the VI.
+ * Eventually the RDP will flush and complete its work as well.
+ * The system will recover after a reset or power cycle.
+ * 
+ */
+void die();
```

Comment:
I think the correct C syntax is `void die(void)`. 

`void die();` is an old C syntax to signal that the argument list is "unknown", so the compiler will accept any potential list of argument. This is often confusing because in C++, instead, `void die()` means "no arguments accepted. But since this is a C declaration, we must follow the C convention.

You can try calling `die(1,2,3);` and it should successfully compile with this declaration.